### ISSUE TITLE
npkill exits unexpectedly on Windows 10 before it finishes loading the folders sizes.

### ISSUE THREAD

**Issue by joseconstela** [voidcosmos/npkill#35]
**Describe the bug**
npkill exits unexpectedly on Windows 10 before it finishes loading the folders sizes.

```bash
\Jose\AppData\Roaming\npm-cache\_npx\35824\node_modules\npkill\lib\services\windows-files.service.js:18
throw err;                                                                                                                                                                                                                                                                                  
^
[Error: UNKNOWN: unknown error, lstat 'C:\Users\Jose\git\<hidden name>\<hidden name>\<hidden name>\<hidden name>\node_modules\.bin\uuid'] {
    errno: -4094,
    code: 'UNKNOWN',
    syscall: 'lstat',
    path: 'C:\\Users\\Jose\\git\\<hidden-name>\\<hidden name>\\<hidden name>\\hidden-name\\node_modules\\.bin\\uuid'
} 
```

**To Reproduce**
1. Open a powershell
2. Run `npx npkill`
3. Wait while npkill calculates folder sizes
4. Before it finishes caculating the sizes, the process exits unaxpectedly, showing an error

**Expected behavior**
npkill will continue to calculate folder sizes and let me to remove them

 - OS: Windows 10 Pro. 10.0.18362
 - Version v0.4.3

**Additional context**
Nothing to add. Feel free to ask for any extra details.

**Comment by zaldih** [comment]
Trying to replicate the error, I have not been able to find that one, but I have discovered others that may be related and I will fix it soon.
Anyway,
Could you go to the indicated file (\ .bin \ uuid) and give me information about it? Does the file have any special permissions? Is it a symbolic link?

Of course, thanks for reporting the error!

**Comment by zaldih** [comment]
We have released version 0.5.0. Now those kinds of exceptions should be controlled.
Although probably if the file has restrictive permissions, or was a broken link, the size of your entire directory is displayed as "--"
